consumergroup.go

```diff
@@ -143,6 +147,17 @@ type ConsumerGroupConfig struct {
 	// back to using Logger instead.
 	ErrorLogger Logger
 
+	// ReadTimeout is the network timeout used when communicating with the
+	// consumer group coordinator.  This value should not be too small since
+	// errors communicating with the broker will generally cause a consumer
+	// group rebalance, and it's undesirable that a transient network error
+	// intoduce that overhead.  Similarly, it should not be too large or the
+	// consumer group may be slow to respond to the coordinator failing over
+	// to another broker.
+	//
+	// Default: 5s
+	ReadTimeout time.Duration
```

Comment:
It looks like a few operations are writes instead of reads. Should we maybe only expose one `Timeout` and use `SetDeadline` instead of `SetReadDeadline`?

Reply:
Sure.  I guess I don't fully understand the subtleties of the read v. write deadlines.  Will change to your suggestion.